pkg/apis/kops/validation/instancegroup_test.go

```diff
@@ -221,3 +221,19 @@ func TestValidBootDevice(t *testing.T) {
 		testErrors(t, g.volumeType, errs, g.expected)
 	}
 }
+
+func TestValidateCloudLabels(t *testing.T) {
+	labels := map[string]string{
+		"nonEmptyLabel": "value",
+		"emptyValue":    "",
+		"":              "emptyTag",
+	}
```

Comment:
Please ad a test for empty key.

Reply:
Doesn't this test both empty key (tag) and empty value?

Reply:
You are right, didn't notice.